pulpcore/app/apps.py

```diff
@@ -234,6 +244,33 @@ def _populate_access_policies(sender, apps, verbosity, **kwargs):
                         )
 
 
+def _populate_system_id(sender, apps, verbosity, **kwargs):
+    # circular import avoidance
+    from pulpcore.app.models import SystemID
+
+    if sender.name != "pulpcore.app":
+        return
+
+    num_system_ids = SystemID.objects.count()
+    if num_system_ids == 0:
+        SystemID().save()
+
+
+def _populate_telemetry_periodic_task(sender, apps, verbosity, **kwargs):
+    # circular import avoidance
+    from pulpcore.app.models import TaskSchedule
```

Comment:
Not sure it is needed...
```suggestion
    TaskSchedule = apps.get_model("core", "TaskSchedule")
```